.github/workflows/unit_tests.yml

```diff
@@ -83,6 +85,10 @@ jobs:
         pip install --upgrade pip
         pip --version
         pip install -r requirements.txt
+        if [[ "${{matrix.python}}" == 3.12 ]]; then
```

Comment:
This will also be required for Python versions > 3.12, right?

Can we cut off the minor part (12) and then do a comparison, so we can forget about this?

Reply:
I thought in the end it's even better to check for functionality: so checking if distutils can be imported rather than an explicit version check.